packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -2,6 +2,64 @@
+| [Administrator Roles API third-party admin status update](#administrator-roles-api-third-party-admin-status-update) | May 5, 2021 |
+| [Custom Group Profile Properties now in Early Access](#custom-group-profile-properties-now-in-early-access) | May 5, 2021 |
+| [Domains API is now GA in Preview](#domains-api-is-now-ga-in-preview) | May 5, 2021 |
+| [Domains API supports OAuth 2.0](#domains-api-supports-oauth-2-0) | May 5, 2021 |
+| [Okta Sign-In Widget, version 5.6.0](#okta-sign-in-widget-version-5-6-0) | May 5, 2021 |
+| [Refresh Token Rotation is now GA in Preview](#refresh-token-rotation-is-now-ga-in-preview) | May 5, 2021 |
+| [SAML parameter SessionNotOnOrAfter now available](#saml-parameter-sessionnotonorafter-now-available) | May 5, 2021 |
+| [System Log API SCIM filter expression update](#system-log-api-scim-filter-expression-update) | May 5, 2021 |
+| [Bug fixed in 2021.05.0](#bug-fixed-in-2021-05-0)         | May 5, 2021 |
+
+#### Access-Control-Expose-Headers now available in CORS response
+
+The `Access-Control-Expose-Headers` header is now included in the response to an Okta CORS endpoint call. This header can be accessed by the cross-origin user agent and includes the following possible header names:  `link`, `x-okta-request-id`, `x-rate-limit-limit`, `x-rate-limit-remaining`, and `x-rate-limit-reset`. <!--OKTA-357710-->
+
+#### Administrator Roles API third-party admin status update
+
+The [Administrator Roles API](/docs/reference/api/roles/) has been updated to support third-party admin status for a user or a group without requiring a role assignment. See [Role assignment operations](/docs/reference/api/roles/#role-assignment-operations). <!--OKTA-381780-->
+
+#### Custom Group Profile Properties now in Early Access
+
+The [Groups API](/docs/reference/api/groups/) can now manage [custom group profile properties](/docs/reference/api/groups/#custom-profile-properties) after these properties are added to the Group Profile schema. The [Schemas API](/docs/reference/api/schemas) includes a new [Groups Schema object](/docs/reference/api/schemas/#group-schema-object) and [operations](/docs/reference/api/schemas/#group-schema-operations) that support custom group properties. You can use the Schemas API or the Profile Editor in the Admin Console to manage schema extensions. Custom Group Profile properties provide flexibility to manage the default profile for Okta groups in the Okta Admin Console Profile Editor or through the Schemas API. This new functionality simplifies group management and lets you quickly add, edit, or remove custom profile attributes to groups. <!--OKTA-389897-->
+
+#### Domains API is now GA in Preview
+
+The [Domains API](/docs/reference/api/domains/) is now GA in Preview. This API allows you to customize your Okta org domain name for a seamless branded experience so that your users will see the same base URL for your application. <!--OKTA-386752-->
```

Comment:
```suggestion
The [Domains API](/docs/reference/api/domains/) is now GA in Preview. This API allows you to customize your Okta org domain name for a seamless branded experience so that your users see the same base URL for your application. <!--OKTA-386752-->
```